### PR TITLE
chore: Add test to verify settings panels are collapsed by default

### LINKED ISSUE #21962: Test: settings "collapses panels by default"
## **Summary**
Really small test difference with 9.x, for the settings page we confirm that the panels are collapsed by default in 9.x.

The panels are closed by default in the implementation, but there is no test confirming this behavior.

## **Acceptance Criteria**
* Write a test to ensure the panels are closed by default.

### **Resources**
9.x Settings page

Screen Shot 2022-04-20 at 11 06 09 AM

10.0 Figma design

Screen Shot 2022-04-20 at 11 08 33 AM

┆Issue is synchronized with this [Jira Task](https://cypress-io.atlassian.net/browse/UNIFY-1571) by [Unito](https://www.unito.io)
┆Attachments: <a href="https://cypress-io.atlassian.net/rest/api/2/attachment/content/11802">Screen Shot 2022-04-20 at 11.06.09 AM.png</a> | <a href="https://cypress-io.atlassian.net/rest/api/2/attachment/content/11803">Screen Shot 2022-04-20 at 11.08.33 AM.png</a>
┆author: Mark Noonan
┆epic: Testing / Quality
┆friendlyId: UNIFY-1571
┆priority: Low
┆sprint: Backlog
┆taskType: Task

### PR DESCRIPTION
**Pull request by astone123**
- Closes #21962

### User facing changelog

n/a

### Additional details

This PR adds a test to ensure that the panels in the app settings are collapsed by default.

See issue for details

### Steps to test

1. In `SettingsCard.vue`, add `initially-open` prop to the `Collapsible` component.
2. Run `SettingsContainer.cy.tsx` and verify that the test fails
3. Remove the prop that you added 
4. Run the tests again and verify that they pass

### How has the user experience changed?

n/a

### PR Tasks

- [x] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Review by ZachJW34 — APPROVED** [review]

File: packages/app/src/settings/SettingsContainer.cy.tsx [hunk 1]
```diff
@@ -5,8 +5,12 @@ import SettingsContainer from './SettingsContainer.vue'
 describe('<SettingsContainer />', { viewportHeight: 800, viewportWidth: 900 }, () => {
   const mountSettingsContainer = () => cy.mountFragment(SettingsContainerFragmentDoc, { render: (gql) => <SettingsContainer gql={gql} /> })
 
-  it('renders', () => {
+  it('renders sections collapsed by default', () => {
     mountSettingsContainer()
```
Comment (mike-plummer) [inline packages/app/src/settings/SettingsContainer.cy.tsx:9]: nitpick: I know you didn't write this, but is there any reason we couldn't restructure `mountSettingsContainer` into a `beforeEach` so it doesn't have to get manually called in each test?

**Comment by marktnoonan** [comment]
Looks good to me - just needs an e2e team reviewer. Adding @rachelruderman.

File: packages/app/src/settings/SettingsContainer.cy.tsx [hunk 2]
```diff
@@ -2,17 +2,19 @@ import { SettingsContainerFragmentDoc } from '../generated/graphql-test'
 import { defaultMessages } from '@cy/i18n'
 import SettingsContainer from './SettingsContainer.vue'
 
+const mountSettingsContainer = () => cy.mountFragment(SettingsContainerFragmentDoc, { render: (gql) => <SettingsContainer gql={gql} /> })
+
+beforeEach(() => mountSettingsContainer())
```
Comment (rachelruderman) [inline packages/app/src/settings/SettingsContainer.cy.tsx:7]: Nice clean up!

File: packages/app/src/settings/SettingsContainer.cy.tsx [hunk 3]
```diff
@@ -2,17 +2,19 @@ import { SettingsContainerFragmentDoc } from '../generated/graphql-test'
...
+
 describe('<SettingsContainer />', { viewportHeight: 800, viewportWidth: 900 }, () => {
-  const mountSettingsContainer = () => cy.mountFragment(SettingsContainerFragmentDoc, { render: (gql) => <SettingsContainer gql={gql} /> })
+  it('renders sections collapsed by default', () => {
+    cy.get('[data-cy="settings"]').should('be.visible')
+    cy.get('[data-cy="setting-expanded-container"]').should('not.exist')
```
Comment (rachelruderman) [inline packages/app/src/settings/SettingsContainer.cy.tsx:12]: Totally up to you, personal preference
```suggestion
    cy.findByTestId('settings').should('be.visible')
    cy.findByTestId('setting-expanded-container').should('not.exist')
```

**Review by rachelruderman — APPROVED** [review]

**Pull request merged**